src/Common/src/System/LocalAppContext.cs

```diff
@@ -40,6 +38,17 @@ private static bool GetCachedSwitchValueInternal(string switchName, ref int swit
             return isSwitchEnabled;
         }
 
+        private static void InitDisableCaching()
+        {
+            bool isEnabled;
+            if (AppContext.TryGetSwitch(@"TestSwitch.LocalAppContext.DisableCaching", out isEnabled))
```

Comment:
I was thinking of a private field for the value (instead of having the compiler generate one for you).